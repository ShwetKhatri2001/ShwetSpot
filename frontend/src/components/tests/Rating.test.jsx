import React from 'react';
import renderer from 'react-test-renderer';

import Rating from '../Rating';

it('should render correctly', () => {
  const tree = renderer.create(<Rating />).toJSON();
  expect(tree).toMatchSnapshot();
});
