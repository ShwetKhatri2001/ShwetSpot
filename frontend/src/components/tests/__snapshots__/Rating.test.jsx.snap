// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="rating"
>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "#4bbf73",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "#4bbf73",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "#4bbf73",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "#4bbf73",
        }
      }
    />
  </span>
  <span>
    <i
      className="far fa-star"
      style={
        Object {
          "color": "#4bbf73",
        }
      }
    />
  </span>
  <span />
</div>
`;
